tests/installation/partitioning_warnings.pm

```diff
@@ -114,33 +119,45 @@ sub run {
     if (!is_storage_ng) {
         # No further warnings on x86_64 and s390x on non-storage-ng
         if (get_var('ARCH') =~ /x86_64|s390x/) {
+            record_info('Test: Snapshots + small root', 'Enable snapshots for undersized root partition');
             process_warning(warning => 'too-small-for-snapshots');
+            record_info('Test: No swap', 'Missing swap partition');
             process_warning(warning => 'no-swap', key => 'alt-n');
         }
         else {
-            process_warning(warning => 'too-small-for-snapshots', key => 'alt-n') if !check_var('ARCH', 'ppc64le');
+            if (!check_var('ARCH', 'ppc64le')) {
+                record_info('Test: Snapshots + small root', 'Enable snapshots for undersized root partition');
+                process_warning(warning => 'too-small-for-snapshots', key => 'alt-n');
+            }
+            record_info('Test: No boot', "Missing boot partition for " . get_var('ARCH'));
             process_missing_special_partitions;
-            process_warning(warning => 'too-small-for-snapshots', key => 'alt-n') if check_var('ARCH', 'ppc64le');
+            if (check_var('ARCH', 'ppc64le')) {
+                record_info('Test: Snapshots + small root', 'Enable snapshots for undersized root partition');
+                process_warning(warning => 'too-small-for-snapshots', key => 'alt-n');
+            }
+            record_info('Test: No swap', 'Missing swap partition');
             process_warning(warning => 'no-swap', key => 'alt-n');
         }
     }
     else {
         # in storage-ng need to process only special warnings
+        record_info('Test: No boot', "Missing boot partition for " . get_var('ARCH'));
         process_missing_special_partitions;
     }
 
-    ## Add wrong or small boot partition
     if (is_storage_ng) {
+        ## Test whether /boot is big enough to contain kernel
+        addpart(role => 'OS', size => 40, format => 'ext2', mount => '/boot');
+        record_info('Test: /boot space', "Boot partition size is insufficient to fit kernel");
+        send_key $cmd{accept};
+        process_warning(warning => 'no-space-for-kernel', key => 'alt-o');
+        remove_partition;
+
+        ## Add wrong or small boot partition
         foreach (keys %roles) {
```

Comment:
Ok, missed that in VR, this introduces weird behavior, as we test boot/zipl and ofw on x86_64, which is non-sense. I will add this to the follow up ticket we have in the next sprint.

Reply:
In fact, that was an intention to check if the partitioner will trigger a warning when there is wrong boot partition.

Reply:
You mean that we create /boot/zipl on x86_64 and check if there is still warning about bios_boot? I guess it's a bit of overkill, considering stability and scalability of our GUI tests =(

Reply:
Technically, user should be warned when the boot partition does not match the arch. Storage-ng checks the size for all boot partition IDs and for some it checks either ID or mount point or file system, depending on what kind of arch we are running the installer. Also the warning message is still the same, so from the point of stability, we are expecting the same needle to appear for each boot partition ID on tested architecture.